lago/verify_configure.py.bck

```diff
@@ -0,0 +1,293 @@
+#
+# Copyright 2014 Red Hat, Inc.
```

Comment:
I guess that this file and all the others below are not needed?

Reply:
No need
I removed it later not to upload it in my local .gitignore